tests/testthat/test-learn.R

```diff
@@ -1,49 +1,87 @@
 # FIXME: #313: learn only from current source
 
-test_that("Learning from MSSQL works?", {
-
+test_that("Standard learning from MSSQL (schema 'dbo') works?", {
 
+  skip_if_src_not("mssql")
   # dm_learn_from_mssql() --------------------------------------------------
-  src_mssql <- skip_if_error(src_test("mssql"))
+  src_mssql <- my_test_src()
 
   # create an object on the MSSQL-DB that can be learned
   if (!any(src_tbls(src_mssql) %>%
     grepl("^tf_1_", .))) {
     copy_dm_to(src_mssql, dm_for_filter(), temporary = FALSE, table_names = unique_db_table_name)
```

Comment:
- Ensure qualified identifiers are not quoted
- Use copied dm to query actual table names for a more robust test

```suggestion
    dm_for_filter_copied <- copy_dm_to(src_mssql, dm_for_filter(), temporary = FALSE, table_names = ~ DBI::SQL(unique_db_table_name(.x)))
```